codespell_lib/data/dictionary.txt

```diff
@@ -1296,28 +1296,42 @@ awefully->awfully
 awnser->answer
 awya->away
 axises->axes
+bacause->because
 baceause->because
+bacic->basic
+backbround->background
 backgorund->background
 backgroud->background
 backrounds->backgrounds
+backsapce->backspace
 backslashs->backslashes
 backupped->backed-up
+backwords->backwards
+backwrad->backward
 bacup->backup
+bage->bag
 bahaving->behaving
 bahavior->behavior
 bahavioral->behavioral
 bahaviors->behaviors
+baisc->basic
 bakc->back
 bakup->backup
 bakups->backups
+balacing->balancing
 baloon->balloon
 baloons->balloons
+balse->false
 banannas->bananas
+bandaid->band-aid
```

Comment:
I feel like we should allow bandaid as an alternative spelling of band-aid.

Reply:
i'm fine either way.

Reply:
done